keyvi/tests/keyvi/index/index_limits_test.cpp

```diff
@@ -28,6 +28,8 @@
 #else
 #include <sys/resource.h>
 
+#include <string>
```

Comment:
why is it needed?

Reply:
`string` is used on line 39. However it obviously worked before, because it is implicitly included from on of the other includes.

The reason why this pops up now: The code is checked with [`cpplint`](https://github.com/cpplint/cpplint), a static code checker. It produces an error called "include what you use". cpplint _must_ pass as part of the checkstyle workflow run on the PR. checkstyle runs _only_ on files that change. So with other words: Because I made a change to this file, cpplint forced me to cleanup.